src/main/java/de/unistuttgart/towercrushbackend/service/websockets/GameService.java

```diff
@@ -205,20 +211,52 @@ public void startTask() {
                         teamWon = true;
                     }
                     final UpdateGameMessage updateGameMessage = new UpdateGameMessage(game);
-                    final MessageWrapper updateLobbyMassageWrapped = websocketService.wrapMessage(
-                        updateGameMessage,
-                        Purpose.UPDATE_GAME_MESSAGE
-                    );
-                    simpMessagingTemplate.convertAndSend(GameService.LOBBY_TOPIC + game.getLobbyName(), updateLobbyMassageWrapped);
+                    final MessageWrapper updateLobbyMassageWrapped;
+                    try {
+                        updateLobbyMassageWrapped = websocketService.wrapMessage(
+                            updateGameMessage,
+                            Purpose.UPDATE_GAME_MESSAGE
+                        );
+                        simpMessagingTemplate.convertAndSend(GameService.LOBBY_TOPIC + game.getLobbyName(), updateLobbyMassageWrapped);
+                    } catch (final JsonProcessingException e) {
+                        log.error("could not parse the message, therefore the frontend could not be informed! error: ", e);
+                    }
                     if (teamWon) {
                         log.info("remove game");
-                        gameRepository.save(this.games.get(game.getLobbyName()));
+                        try {
+                            gameRepository.save(this.games.get(game.getLobbyName()));
+                        } catch (final Exception e) {
+                            log.info("Game results were null: " + e);
+                            timeUpdate.cancel(true);
+                            timeUpdate = null;
+                        }
                         this.games.remove(game.getLobbyName());
                         teamWon = false;
                     }
                 }
-                Thread.sleep(1000);
+                final int sleepTime = 1000;
+                try {
+                    Thread.sleep(1000);
```

Comment:
```suggestion
                final int sleepTime = 1000;
                try {
                    Thread.sleep(sleepTime);
```

use the variable you set